Libraries/ART/React-ART.podspec

```diff
@@ -23,7 +23,7 @@ Pod::Spec.new do |s|
   s.homepage               = "https://reactnative.dev/"
   s.license                = package["license"]
   s.author                 = "Facebook, Inc. and its affiliates"
-  s.platforms              = { :ios => "10.0", :tvos => "10.0", :osx => "10.13" } # TODO(macOS GH#214)
+  s.platforms              = { :ios => "10.0", :tvos => "10.0", :osx => "10.15" } # TODO(macOS GH#214)
```

Comment:
Am I misunderstanding or does this change the minimum supported OS to 10.15 (it should be 10.14 and that is what you mention in the README

Reply:
yes, Anand is correct.